pkgs/servers/nosql/mongodb/v4_4.nix

```diff
@@ -10,5 +10,10 @@ buildMongoDB {
   sha256 = "sha256-ebg3R6P+tjRvizDzsl7mZzhTfqIaRJPfHBu0IfRvtS8=";
   patches = [
     ./forget-build-dependencies-4-4.patch
+   (fetchpatch {
+      name = "mongodb-4.4.1-gcc11.patch";
+      url = "https://raw.githubusercontent.com/gentoo/gentoo/7168257cad6ea7c4856b01c5703d0ed5b764367c/dev-db/mongodb/files/mongodb-4.4.1-gcc11.patch";
```

Comment:
Do you prefer this commit from MongoDB dev ? [https://github.com/mongodb/mongo/commit/e78b2bf6eaa0c43bd76dbb841add167b443d2bb0](https://github.com/mongodb/mongo/commit/e78b2bf6eaa0c43bd76dbb841add167b443d2bb0.patch) same for 4.2 and 4.0

Reply:
I did a bit of archeology to find those.

Reply:
I used this patch in #178820 maybe you can cherry-pick my commit inside your branch.

Reply:
Sounds nice too, if it builds it builds!

Did you fully built it and run with https://github.com/NixOS/nixpkgs/pull/178820? 

And just for the records, you used both https://github.com/mongodb/mongo/commit/107858d4d2ffd9f0690def5108898a8b4be70350 and https://github.com/mongodb/mongo/commit/e78b2bf6eaa0c43bd76dbb841add167b443d2bb0?

Reply:
I added in flag "install-mongod" so it fully built only mongod and mongos. I might be mistaken but test where built but not executed. And I focused on 5.0 (I don't need to do migration) and only second patch was needed.

Reply:
Yes, please prefer fetching from upstream instead.